### DISCUSSION TITLE
Options for how season packs are searched for

### DISCUSSION THREAD

**Discussion by Xander567** [pymedusa/Medusa#7589] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
When searching for a season, Medusa appens the number and the word season on the end. ie

Steins Gate 0 Season

I confirmed this in Jackett. Most providers seem ok with this and will return results. I have one site that returns nothing for this, and needs the search to be less specific. 

**Describe the solution you'd like**
An option on a tv show (or provider) that lets you either define the search terms, or a checkbox that results in less specific searching to be done (search by just name only). Season/episodes etc can be done in post processing. 

**Describe alternatives you've considered**
Ditching the site and using one that doesn't have such a sucky search engine. 

**Additional context**
An alternative could be to have the option on the provider instead of the tv-show. This could result in less data coming back that needs to be post processed.

**Comment by p0psicles** [comment]
Im working on a feature where you can specify your search terms using template tags. I could also include season searches in this.

**Comment by p0psicles** [comment]
#3732

**Comment by Xander567** [comment]
Just had a look at the issue, and it seems to do what i am wanting. Look forward to it being added.
